.env

```diff
@@ -19,7 +19,7 @@ APP_ENV=dev
 APP_SECRET=ad7df41862a7f4faa9485d384b8388ef
 PROFILE_PICTURES_DIRECTORY=public/uploads/profile_pictures
 REGISTRATION_BASE_COST=10000
-API_HOST=http://dev.symfony-6-testing-project.com
+API_HOST=http://dev.symfony-6-testing-project.com:84
```

Comment:
So now your internal call to api will not work because of :84. Revert this and add new env variable APP_HOST with :84 there, and use it for email